extras/cloudshell/tutorial.md

```diff
@@ -9,44 +9,49 @@ You'll be able to run Bank of Anthos on:
 
 Let's get started!
 
-## Kubernetes cluster setup
+## Select a GCP project
 
-Set up a Kubernetes cluster using the instructions below for either **minikube** or **GKE**.
+In order to create a Google Kubernetes Engine cluster, you'll need to create a Google Cloud Platform project or use an existing project.
 
----
-### Minikube
+Note: If you choose to deploy to minikube instead of GKE, you can skip this step.
 
-Minikube creates a local Kubernetes cluster on Cloud Shell.
+<walkthrough-project-setup></walkthrough-project-setup>
 
-1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+## Kubernetes cluster setup
 
-2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+This section shows you how to set up a Kubernetes cluster using either **minikube** or **GKE**.
 
-3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+1. Navigate to the <walkthrough-editor-spotlight spotlightId="cloud-code-k8s-explorer">**Clusters**</walkthrough-editor-spotlight> explorer in the <walkthrough-editor-spotlight spotlightId="activity-bar-cloud-k8s">Cloud Code - Kubernetes sidebar</walkthrough-editor-spotlight> using the left side Activity bar.
```

Comment:
I'd reword it so the first spotlight that shows is the first one to be clicked. Currently they're in reverse order. e.g. something along the lines of "in the **cloud code - kubernetes** tab of the activity bar, navigate to the **clusters** explorer"

Or alternatively (I like this one a bit more), this bullet point could just be to "navigate to the **cloud code - kubernetes** tab" whereas in the next bullet point, we'd prefix with "in the **clusters explorer**, click **+** in the title bar... etc etc"

(spotlights in bold)

Reply:
I like your alternate option, to not just fix the order but to move the "Clusters explorer" to the next bullet point! Updated ✅